google/cloud/bigtable/instance.py

```diff
@@ -644,6 +644,184 @@ def list_tables(self):
+
+        				The fields eligible for filtering are:
+
+                -  ``name``
+                -  ``source_table``
+                -  ``state``
+                -  ``start_time`` (values of the format YYYY-MM-DDTHH:MM:SSZ)
+                -  ``end_time`` (values of the format YYYY-MM-DDTHH:MM:SSZ)
+                -  ``expire_time`` (values of the format YYYY-MM-DDTHH:MM:SSZ)
+                -  ``size_bytes``
+
+                		To filter on multiple expressions, provide each
+                		separate expression within parentheses. By default,
+                		each expression is an AND expression. However, you can
+                		include AND, OR, and NOT expressions explicitly.
+
+                		Some examples of using filters are:
+
+                -  ``name:"exact"`` --> The Backup name is the string "exact".
+                -  ``name:howl`` --> The Backup name contains the string "howl"
+                -  ``source_table:prod`` --> The source table's name contains
+                		the string "prod".
+                -  ``state:CREATING`` --> The Backup is pending creation.
+                -  ``state:READY`` --> The Backup is created and ready for use.
+                -  ``(name:howl) AND (start_time < \"2020-05-28T14:50:00Z\")``
+                		--> The Backup name contains the string "howl" and
+                		the Backup start time is before 2020-05-28T14:50:00Z.
+                -  ``size_bytes > 10000000000`` --> The Backup size is greater
+                		than 10GB
+
+        :type order_by: str
+        :param order_by: (Optional) An expression for specifying the sort order
+        				 of the results of the request. The string value should
+        				 specify one or more fields in ``Backup``. The full
+        				 syntax is described at https://aip.dev/132#ordering.
+
+                		 Fields supported are: \* name \* source\_table \*
+                		 expire\_time \* start\_time \* end\_time \*
+                		 size\_bytes \* state
+
+                		 For example, "start\_time". The default sorting order
+                		 is ascending. To specify descending order for the
+                		 field, a suffix " desc" should be appended to the
+                		 field name. For example, "start\_time desc". Redundant
+                		 space characters in the syntax are insigificant. If
+                		 order_by is empty, results will be sorted by
+                		 ``start_time`` in descending order starting from
+                		 the most recently created backup.
+
+        :type page_size: int
+        :param page_size: (Optional) The maximum number of resources contained
+        				  in the underlying API response. If page streaming is
+        				  performed per-resource, this parameter does not
+        				  affect the return value. If page streaming is
+        				  performed per-page, this determines the maximum
+        				  number of resources in a page.
```

Comment:
Wrong indents in these docstrings

Reply:
This is strange! The code was blackened, looks fine in my PyCharm, tabs disabled, but here is totally skewed. I'm baffled. Well, let's try another commit then.